Sources/PointFree/Account/Account.swift

```diff
@@ -426,61 +426,86 @@ public func nextBilling(for subscription: Stripe.Subscription) -> String {
   }
 }
 
-private let subscriptionPlanRows = View<Stripe.Subscription> { subscription in
-  return div([`class`([Class.padding([.mobile: [.top: 1, .bottom: 3]])])], [
-    gridRow([
+private let subscriptionPlanRows = View<Stripe.Subscription> { subscription -> Node in
+
+  let planRow = gridRow([
```

Comment:
broke out some rows of this part of the account page into variables so that I could assemble them more easily.